h3d/scene/Graphics.hx

```diff
@@ -221,6 +221,13 @@ class Graphics extends Mesh {
 		setColor(color, alpha);
 	}
 
+	public function setColorF( r : Float, g : Float, b : Float, a : Float ) {

```

Comment:
```suggestion
	public function setColorF( r : Float, g : Float, b : Float, a : Float = 1. ) {
```

I think it would be a good idea to make alpha value optional with default value of 1 to keep in tune with other color setting methods (`setColor` / `lineStyle`).

Reply:
Updated.

Reply:
Anychance someone could approve and merge?